test/Serialization/module_package_name.swift

```diff
@@ -0,0 +1,15 @@
+// RUN: %empty-directory(%t)
+// RUN: %{python} %utils/split_file.py -o %t %s
```

Comment:
For the next time, there's a lit command to split a test into multiple source files now:
```
// RUN: split-file %s %t

//--- Lib.swift
//--- Client.swift
```